### PR TITLE
"/watch add" embed instead of full content

### PR DESCRIPTION
**Pull request by itsWindows11**

### CODE REVIEW

**Comment by itsWindows11** [comment]
Bruh why does it get past commits that have been already merged?

**Comment by Jasius** [comment]
> Bruh why does it get past commits that have been already merged?

Most likely due to previous changed being squashed rather than merged entirely, I'd suggest you to create a clean branch off Dotsimus main branch for the future PRs.

File: README.md [hunk 1]
```diff
@@ -24,9 +24,9 @@ Machine learning powered moderation brings misbehaving, toxic users to moderator
 ## Track topics, names or keywords,stay in the loop of things!
 Keywords tracking will make sure that you know whenever you're mentioned and missed! It can also be used to track topics that you find interesting.
 
-`!track \<word>`
+`/watch remove <word>`
```
Comment (Jasius) [inline README.md:27]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -24,9 +24,9 @@ Machine learning powered moderation brings misbehaving, toxic users to moderator
...
   
-`!watch \<word>`
+`/watch add <word>`
```
Comment (Jasius) [inline README.md:29]:
```suggestion
`/watch add`
```

**Review by Jasius — APPROVED** [review]

**Pull request merged**